sys/arduino/doc.txt

```diff
@@ -64,6 +64,8 @@
  * [Makefile.include](https://github.com/RIOT-OS/RIOT/tree/master/sys/arduino/Makefile.include)
  * of the RIOT Arduino module.
  *
+ * TODO UPDATE DOCUMENTATION
```

Comment:
would be very much appreciated 😄

Reply:
Yeah, thats why I put this line either I could forget.